### PR TITLE
Remove virtualenv activation #60

### PR DESCRIPTION
**Pull request by rafatower**
Please take a look at #60 for the discussion of whether to use or not virtualenvs from the extension.

@jgoizueta please review
 
cc/ @luisbosque @rochoa

### CODE REVIEW

**Comment by jgoizueta** [comment]
seems good to me

**Comment by rafatower** [comment]
@jgoizueta I added a couple more commits, the important one being the one touching Makefiles

File: Makefile [hunk 1]
```diff
@@ -52,10 +50,6 @@ clean-release: ## clean up current release
 	rm -rf release/python/$(RELEASE_VERSION)
 	rm -f release/$(RELEASE_VERSION)--*.sql
 
-# Cleanup all virtual environments
-clean-environments: ## clean up all virtual environments
-	rm -rf envs/*
-
 clean-all: clean-dev clean-release clean-environments
```
Comment (jgoizueta) [inline Makefile]: clean-environments reference must be removed from clean-all

File: src/py/Makefile [hunk 2]
```diff
@@ -2,21 +2,16 @@ include ../../Makefile.global
 
 # Install the package locally for development
 install:
-	virtualenv --system-site-packages ../../envs/dev
-	# source ../../envs/dev/bin/activate
-	../../envs/dev/bin/pip install -I ./crankshaft
-	../../envs/dev/bin/pip install -I nose
```
Comment (jgoizueta) [inline src/py/Makefile]: don't we need to install `nose` for development?

File: Makefile (hunk 1 above)
Comment (rafatower) [inline Makefile]: thanks, done

**Pull request merged**